src/test/java/cloud/fogbow/ras/core/plugins/interoperability/openstack/securityrule/v2/OpenStackSecurityRulesPluginTest.java

```diff
@@ -1,258 +1,512 @@
+        DatabaseManager.class,
+        GetSecurityGroupsResponse.class,
+        GetSecurityRulesResponse.class,
+        OpenStackHttpToFogbowExceptionMapper.class,
+        PropertiesUtil.class,
+})
+public class OpenStackSecurityRulesPluginTest extends BaseUnitTests {
+
+    private static final int FAKE_PORT_FROM = 1024;
+    private static final int FAKE_PORT_TO = 2048;
+    
+    private static final String ANY_STRING = "any-string";
+    private static final String ANY_URL = "http://localhost:8007";
     private static final String NETWORK_NEUTRONV2_URL_KEY = "openstack_neutron_v2_url";
     private static final String DEFAULT_NETWORK_URL = "http://localhost:0000";
+    private static final String INGRESS_TRAFFIC = "ingress";
     private static final String SECURITY_RULE_ID = "securityRuleId";
     private static final String SECURITY_GROUP_ID = "securityGroupId";
+    private static final String SECURITY_GROUP_NAME = "securityGroupName";
 
-    private static final String FAKE_TOKEN_PROVIDER = "fake-token-provider";
+    private static final String FAKE_ETHERTYPE = "IPv4";
     private static final String FAKE_TOKEN_VALUE = "fake-token-value";
     private static final String FAKE_USER_ID = "fake-user-id";
     private static final String FAKE_NAME = "fake-name";
-    private static final String FAKE_CLOUD_NAME = "fake-cloud-name";
     private static final String FAKE_PROJECT_ID = "fake-project-id";
-    private static final String FAKE_USER_NAME = "fake-user-name";
-    private static final String FAKE_MEMBER_ID = "fake-member-id";
-    private static final String FAKE_GATEWAY = "fake-gateway";
-    private static final String FAKE_ADDRESS = "fake-address";
-
-    private OpenStackSecurityRulePlugin openStackSecurityRulePlugin;
-    private OpenStackV3User openStackV3Token;
-    private HttpRequestClient clientUtil;
+    private static final String MAP_METHOD = "map";
+    private static final String PROTOCOL_ANY = "ANY";
+    private static final String PROTOCOL_ICMP = "ICMP";
+    private static final String PROTOCOL_TCP = "TCP";
+    private static final String PROTOCOL_UDP = "UDP";
+
+
+    private OpenStackSecurityRulePlugin plugin;
+    private OpenStackV3User cloudUser;
     private Properties properties;
-    private OpenStackHttpClient openStackHttpClient;
+    private OpenStackHttpClient clientMock;
+    private NetworkOrder majorOrder;
 
     @Before
     public void setUp() throws InvalidParameterException, UnexpectedException {
-        PropertiesHolder propertiesHolder = PropertiesHolder.getInstance();
-        this.properties = propertiesHolder.getProperties();
-        this.properties.put(NETWORK_NEUTRONV2_URL_KEY, DEFAULT_NETWORK_URL);
-
+        this.testUtils.mockReadOrdersFromDataBase();
         String confFilePath = HomeDir.getPath() + SystemConstants.CLOUDS_CONFIGURATION_DIRECTORY_NAME + File.separator
                 + "default" + File.separator + SystemConstants.CLOUD_SPECIFICITY_CONF_FILE_NAME;
```

Comment:
Use the constant DEFAULT_CLOUD_NAME  from TestUtils class.

Reply:
done.